### DISCUSSION TITLE
Does this work in Firefox 125 on macOS?

### DISCUSSION THREAD

**Discussion by MoltenMonster** [xiaoxiaoflood/firefox-scripts#302] [General Support]
I followed all the instructions but it doesn't seem to work, I downloaded the following scripts from here https://github.com/xiaoxiaoflood/firefox-scripts:
- BeQuiet: https://raw.githubusercontent.com/xiaoxiaoflood/firefox-scripts/master/chrome/BeQuiet.uc.js
- Enter Selects: https://raw.githubusercontent.com/xiaoxiaoflood/firefox-scripts/master/chrome/enterSelects.uc.js
- rebuild_userChrome.uc.js
> If you want a button to manage your scripts, including the ability to disable/enable scripts without needing to restart¹ Firefox or Thunderbird, save [rebuild_userChrome.uc.js](https://raw.githubusercontent.com/xiaoxiaoflood/firefox-scripts/master/chrome/rebuild_userChrome.uc.js) into chrome.

**Comment by 117649** [comment]
https://onemen.github.io/tabmixplus-docs/other/installation/

**Reply by MoltenMonster** [reply]
Thanks, it turns out I extracted the files right into the chrome folder instead of into the `chrome/utils` folder.
